<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatureMerge</class>
 <widget class="QWidget" name="FeatureMerge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Feature Merge</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,1,2" columnminimumwidth="1,2,1,2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Feature Matrix 1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditFeature1Path"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="buttonLoadFeatureMatrix1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="2">
        <widget class="QTextEdit" name="textEditDescription">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Adobe Devanagari'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:6pt;&quot;&gt;DESCRIPTION: Make sure the case ID is in the first columne.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Feature Matrix 2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditFeature2Path"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="buttonLoadFeatureMatrix2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QTableWidget" name="tableFeature1"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Feature Matrix 1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTableWidget" name="tableFeature2"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Feature Matrix 2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,2,3" columnminimumwidth="1,2,3">
       <item row="0" column="0">
        <widget class="QRadioButton" name="radioMergeTypeFeatures">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Merge Feature</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QRadioButton" name="radioMergeTypeCases">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Merge Case</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Add PreName in Feature Matrix 1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditPreName1"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Add PreName in Feature Matrix 2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditPreName2"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="buttonMerge">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Merge</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
